Exercise7.js

```diff
@@ -0,0 +1,2 @@
+const multiplyNumbers = (x,y) =>  x*y;
```

Comment:
Add some spaces to make the code more readable
ex: `const multiplyNumbers = (x, y) =>  x * y;`